includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
Your option is also valid. You need to decide if this limitation (you have to display lang dropdown if you want to put wiki there) won't be painful in future.